src/library/scala/collection/immutable/BitSet.scala

```diff
@@ -137,7 +139,7 @@ object BitSet extends BitSetFactory[BitSet] {
         if (elems1 == 0L) throw new NoSuchElementException("Empty BitSet")
         new BitSet2(elems0, elems1 - java.lang.Long.lowestOneBit(elems1))
```

Comment:
i think this one should use `createSmall`, as `elems1 - java.lang.Long.lowestOneBit(elems1)` may be `0L`.